plugins/jss-isolate/simple/style.js

```diff
@@ -0,0 +1,12 @@
+(function() {
+  window.styles = {
+    'button': {
+      'font-size': '20px',
```

Comment:
why do you define font-size here? The point of the plugin is to see it automatically resetted, right?